site/docs/components/slider.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Slider
+navTitle: Slider (not built)
+summaryParagraph: A slider lets people imprecisely select a value from a subjective range by moving a handle horizontally.
+tags: ["Range", "Range input", "Range slider"]
+needToKnow:
+- Avoid sliders when choosing a precise value is important.
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### Examples
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D47508%253A35978&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+Sliders are helpful for choosing subjective values.
+
+- Always show the current, selected value.
+- Provide a visible label describing what the slider changes, such as "Brightness".
+- Show the minimum value on the left and maximum value on the right, where these are reversed in Right-To-Left (RTL) languages.
+- We only use horizontal sliders. We're yet to need a vertical slider.
+- We only use sliders with no fill so far.
```

Comment:
The old performance slider looks to have that purple fill — is that an exception or deprecation?

Reply:
It's a good use case to call out! Made some tweaks